app/src/main/java/com/zulip/android/AsyncStatusUpdate.java

```diff
@@ -83,8 +83,6 @@ private JSONObject chooseLatestPresence(JSONObject person)
 
     @Override
     protected void onPostExecute(String result) {
-        super.onPostExecute(result);
-
```

Comment:
Would cleanup the commit message to not have such a long first line but lgtm assuming you've tested this.